packs/equipment-effects/effect-five-feather-wreath-weapon.json

```diff
@@ -0,0 +1,144 @@
+{
+    "_id": "NPeuNp3C6Wq94Oub",
+    "img": "systems/pf2e/icons/equipment/other/spellhearts/five-feather-wreath.webp",
+    "name": "Effect: Five-Feather Wreath - Weapon",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Five-Feather Wreath], @UUID[Compendium.pf2e.equipment-srd.Item.Five-Feather Wreath (Greater)], @UUID[Compendium.pf2e.equipment-srd.Item.Five-Feather Wreath (Major)]</p>\n<p>You can Fly 5 feet as a single action, or you can Fly a number of feet as a free action if the spell you cast wasn't a cantrip. This must be your next action, and if you don't have a fly Speed, you must end the flight on solid ground or fall.</p>"
```

Comment:
Not sure we need an effect for this, since triggering the wreath does what it says immediately?

Reply:
Same goes for the armor effect.

Hmm

Suppose we could forgo both effects and just add the automation to the item itself? We could add a choice set roll option that kicks in to allow one to select weapon or armor and use that to determine which effect goes off.

Thoughts?

Reply:
The armor effect does grant a bonus rather than just a reminder note, and it does last indefinitely.